mesonbuild/compilers/mixins/visualstudio.py

```diff
@@ -137,6 +138,11 @@ def __init__(self, target: str):
 
     # Override CCompiler.get_always_args
     def get_always_args(self) -> T.List[str]:
+        if not isinstance(self, ClangClCompiler) and mesonlib.version_compare(self.version, '<19.00'):
+            try:
+                self.always_args.remove('/utf-8')
```

Comment:
Why do we do this for any compiler other than clang-cl before 19.00? Are we sure there aren't any other compilers that are visual studio "like", but use different versioning schemes that happen to be <19?

Reply:
Hi @eli-schwartz,

Ah, thank you!  Should have thought of this...